bigtable/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/google/btree v1.0.1
 	github.com/google/go-cmp v0.5.6
 	github.com/syndtr/goleveldb v1.0.0
+	github.com/testcontainers/testcontainers-go v0.11.1 // indirect
```

Comment:
Okay... having played with this for a bit, I'm not sure I love this.  I like the containers idea, but I have a couple of issues with this formulation:

- it introduces additional dependencies for anyone using this lib
- you can't actually just run the normal tests without building the containers-- it forces a dependency on docker

I would create a separate go module, sibling to bigtable and storage, that imports the other two.  And put your container testing examples there.

Reply:
I agree with you, I added it mainly to test.

What if we leave an example in the readme?

Reply:
Another option is to add a new folder (different go.mod) only with tests and then testcontainers would be imported only there

Reply:
^^ yes, that is what I had in mind!